definitions/json/ComplexTypes/ComplexTypes.json

```diff
@@ -6315,14 +6320,16 @@
     "FieldType": "Number",
     "SecurityClassification": "Public"
   },
+
   {
     "ID": "etInitialConsiderationRule27",
     "ListElementCode": "etICRule27ClaimToBe",
     "ElementLabel": "Claim to be",
     "FieldType": "FixedRadioList",
     "RetainHiddenValue": "No",
     "FieldTypeParameter": "fl_rule2728ClaimToBe",
-    "SecurityClassification": "Public"
+    "SecurityClassification": "Public",
+    "Publish": "Y"
```

Comment:
Hi @addison-jephtah 
There is no Publish on ComplexTypes. It might be needed to be added to EventToComplexTypes